Localize preservation event datetimes, refs #13417

replaceafill opened the pull request:
Archivematica uses UTC datetimes for the events in the METS. This
updates the DO metadata component to localize those values on display.

djjuhasz reviewed (approved):
Looks good to me 👍

djjuhasz commented on apps/qubit/modules/digitalobject/actions/metadataComponent.class.php:
```diff
@@ -307,9 +307,13 @@ protected function setOriginalFileShowProperties()
     );
+    // Convert this UTC string property to local time
+    $this->originalFileIngestedAt = $this->localizeUTCDatetime(
+      (string)$this->resource->object->originalFileIngestedAt
```
Most examples of PHP type casting I've seen have a space between the cast and the variable, including in the PHP docs:
https://www.php.net/manual/en/language.types.type-juggling.php#language.types.typecasting

I'd like to standardize on this convention for AtoM

replaceafill replied:
:+1: I also updated another existing type cast in the same template.

djjuhasz commented on apps/qubit/modules/digitalobject/actions/metadataComponent.class.php:
```diff
@@ -351,4 +359,12 @@ protected function setPreservationCopyShowProperties()
     );
   }
 
+  protected function localizeUTCDatetime($dateTime)
```
Nitpick: I think the method name should be `localizeUTCDateTime` (capital on "Time") to match the `$dateTime` variable capitalization, and the built-in [DateTime Class](https://www.php.net/manual/en/class.datetime.php)

djjuhasz reviewed (approved):
Looks good, I just had a couple of nitpicks. :)

replaceafill committed:
Localize preservation event datetimes, refs #13417

Archivematica uses UTC datetimes for the events in the METS. This
updates the DO metadata component to localize those values on display.

replaceafill commented:
Thanks @djjuhasz!
